### ISSUE TITLE
Find a way to test our CLI

### ISSUE THREAD

**Issue by irevoire** [meilisearch/meilisearch#3239]
Currently, our CLI is barely tested.
We've already encountered a bunch of bugs in the past, and we just discovered (two new bugs](https://github.com/meilisearch/meilisearch/issues/3238) after the huge refacto of the v0.30.

It's a shame that we don't have a whole test suite targeting our CLI, but I understand it's hard, and it wasn't prioritized.

I would like to gather some resources on how we could do it to start working on it after the v1.0.
I think that's the only way we can ensure we don't break anything after a refacto.
The config file and the env variables aren't tested either :pensive:

**Comment by curquiza** [comment]
I agree, this is something we should prioritize in our test suite, definitely. Thanks for opening the issue @irevoire
